app/views/applications/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<h1><%= @application.name %>'s Application</h1>
+<p>Address: <%= @application.full_address %></p>
+<p>Description of why I would make a good home: <%= @application.description %></p>
+<p>Application Status: <%= @application.status %></p>
+
+<h2>Pets Applied For: </h2>
+<% @application.list_of_pets.each do |pet| %>
+<p><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></p>
+<% end %>
+
+<% if @application.status == "In Progress" && @application.added_pets? %>
+
+  <h2>Add a Pet to this Application</h2>
+  <%= form_with url: show_application_path, method: :get do |f| %>
+    <%= f.label :pet_name, 'Search for Pets by name:' %>
+    <%= f.text_field :pet_name %>
+    <%= f.submit "Submit" %>
+  <% end %>
+
+  <% if params[:pet_name].present? %>
+    <% @pets.each do |pet| %>
+    <ul>
+      <li><%= pet.name %> <%= button_to "Adopt this pet", create_application_pets_path, method: :patch, params: {'pet_id' => pet.id, 'application_id' => params[:id], 'submit_app' => 'yes'} %></li>
+    </ul>
+    <% end %>
+  <% end %>
+
+  <section id="submit-application">
+    <h2>Submit Application</h2>
+    <%= form_with url: update_application_path, method: :patch, data: { turbo: false } do |f| %>
+    <%= f.label :good_owner_comments, "Why I would make a good owner: " %>
+    <%= f.text_field :good_owner_comments %>
+    <%= f.submit "Submit Application" %>
+  </section>
+  <% end %>
+
+<% elsif @application.status == "Pending" %>
+
+  <p>Why I would make a good owner: <%= @application.good_owner_comments %></p>
+  
+<% else %>
+
+  <section id="add-pet">
```

Comment:
This section should be able to be consolidated so that it's not duplicated in the view. The sections of the page could be restructured like so (pseudocode below):
```
if status is not pending
    - option to search for pets
    - if pets are already present on the app
              -area to display associated pets
              -option to submit
else # i.e. status is pending
     show why applicant would make a good owner



```